Zenventory/Services/CoreDataService.swift

```diff
@@ -63,23 +61,26 @@ class CoreDataService: ObservableObject, CoreDataManager {
         if price != nil {
             newProduct.price = price!
```

Comment:
I would recommend you use `if let` or `guard let` instead, to avoid using force unwrap. I understand that you're checking for the existence of a non- `nill` value, however, SwiftLint will scream at that. And this is just really not the **recommended** way to do it.